dockerfiles/aws-ruby/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.1
+FROM ghcr.io/alphagov/verify/ruby:2.6.6
 
 LABEL ruby="2.6.1"
```

Comment:
Surely the label should match

Reply:
That's the problem with having a vendoring pipeline - it's not great for having specific versions. We are planning to solve this

Reply:
@galund 2.6.6 is a minor point release and doesn't break this.  It also put this on par with all our other repositories which all use Ruby 2.6.6.

Reply:
In which case the label should just be "2.6" - this makes no sense as is, and will just confuse us.